packages/tasks/src/model-libraries.ts

```diff
@@ -907,6 +907,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"soloaudio_v2.pt"`,
 	},
+	"song-generation": {
```

Comment:
haha, `song-generation` is a bit too generic tag no? I wonder if we should change it a bit.